_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+It can get more expensive to add new capabilities in the future.
+High internal quality, on the other side, can lead to faster delivery of new features; simply, because there is less cruft to get in the way {% cite Fowler2019 %}.
+
+Essentially, architecture reviews aim at {% cite Maranzano2005 %}:
+  1. finding software design problems early in development before they get costly.
+  1. building projects based on best practices and transfer this knowledge across the organization.
+  1. improving the organization's software quality and operations (-> documentation).
+
+Therefore, architecture review processes formalize different steps, the parties involved, or input and output to architecture reviews.
+Over the last decades, different architecture review processes have emerged.
+
+As every such process can be modified and adapted to the unique requirements of a specific company, we will focus on three different prototypical processes that are described in literature:
+  1. The "Classical" Architecture Review process explained by Maranzano et al. in {% cite Maranzano2005 %}.
+  1. The Architecture Decision Records initially described by Nygard in {% cite Nygard2011 %}.
+  1. The Lightweight Request for Comment (RFC)/Design Document approach explained in {% cite Ubl2020 %}, {% cite Winters2020 %}, {% cite Orosz2020 %}, {% cite Orosz2021 %}, {% cite Zimmermann2019 %}, {% cite Gonchar2020 %}, and {% cite Mozilla2020 %} from the perspectives of different organizations - sometimes called Lightweight RFC and sometimes Design Document approach. The approach has also been mentioned in the ThoughWorks Tech Radar Vol. 24 trial area as Lightweight RFC approach in {% cite ThoughtWorks2021 %}.
+
+In the next sections, we will describe these three different prototypical architecture review processes.
+
+## The "Classical" Architecture Review Process
+
+The "Classical" Architecture Review process is described by Maranzano et al. in a paper based on common architecture review processes at AT&T, Avaya, Lucent, and Millenium Services {% cite Maranzano2005 %}.
+The paper stems from 2005, the idea is based on AT&T's practices from the late 1980s.
+A lot of established companies (still) use this approach in some variation.
+
+### Parties
+
+The approach considers three primary parties in an architecture review {% cite Maranzano2005 %}:
+  1. The project team which requests the architecture review.
+  1. The review team which consists of experts assembled for the review on the basis of their expertise, their independence from the specific project, and their ability to conduct themselves appropriately in a potentially difficult interpersonal situation.
+  1. The Architecture Review Board (ARB) which is a standing team that oversees the review process and its effect on the organization.
+
+Moreover, there are further roles in the overall process - depending on the size of the enterprise and if these roles are required:
```

Comment:
You are very right. I will limit myself to "organization" here, because it does not always have to be a company or enterprise. It can also be a part of a company or an open source community.